src/filters/events.cjs

```diff
@@ -82,38 +119,35 @@ const getEvents = (collection) => {
 };
 
 /* @docs
-label: isFuture
-category: Upcoming
-note: Check that the page/event has a start date in the future (or today)
-example: |
-  {%- if event | isFuture -%}
-    <strong>{{ utility.datetime(event.date) }}</strong>
-  {%- else -%}
-    {{ utility.datetime(event.date) }}
-  {%- endif %}
-params:
-  page:
-    type: event object
-*/
-const isFuture = (event) =>
-  event.end ? getDate(event.end) >= now() : getDate(event.date) >= now();
-
-/* @docs
-label: getFuture
-category: Upcoming
-note: Return only the pages/events in the future
-example: |
-  {% set events = events | getFuture if events else none %}
+label: birdEvents
+category: List
+note: |
+  Group events by Bird
 params:
   events:
-    type: array of events
+    type: collection of events
 */
-const getFuture = (events) => events.filter(isFuture);
+const birdEvents = (events) => {
```

Comment:
This new filter needs tests. (I can do that tomorrow if you'd prefer not to.)

Reply:
Yeah, if you have time that would be great. I can address the other notes here.